tests/integration/helpers.py

```diff
@@ -490,6 +517,41 @@ async def check_tls(ops_test: OpsTest, unit_name: str, enabled: bool) -> bool:
         return False
 
 
+async def check_tls_patroni_api(ops_test: OpsTest, unit_name: str, enabled: bool) -> bool:
+    """Returns whether TLS is enabled on Patroni REST API.
+
+    Args:
+        ops_test: The ops test framework instance.
+        unit_name: The name of the unit where Patroni is running.
+        enabled: check if TLS is enabled/disabled
+
+    Returns:
+        Whether TLS is enabled/disabled on Patroni REST API.
+    """
+    unit_address = get_unit_address(ops_test, unit_name)
+    tls_ca = await get_tls_ca(ops_test, unit_name)
```

Comment:
Could you return False here `if not tls_ca`?

Reply:
Yes, I'll add it.

Reply:
Adde on https://github.com/canonical/postgresql-operator/pull/33/commits/334a50d1da908dda09563dc10e5eeb70975e8135.